app/src/main/java/io/scalac/degree/data/schedule/filter/ScheduleFilterManager.java

```diff
@@ -0,0 +1,167 @@
+                .equalTo("isActive", true).findAll();
+        realm.close();
+        return result;
+    }
+
+    public List<RealmScheduleTrackItemFilter> getActiveTrackFilters() {
+        final Realm realm = realmProvider.getRealm();
+        final List<RealmScheduleTrackItemFilter> result = realm
+                .where(RealmScheduleTrackItemFilter.class)
+                .equalTo("isActive", true).findAll();
+        realm.close();
+        return result;
+    }
+
+    public void createDayFiltersDefinition(List<ConferenceDay> conferenceDays) {
+        final Realm realm = realmProvider.getRealm();
+        realm.beginTransaction();
+        realm.allObjects(RealmScheduleDayItemFilter.class).clear();
+        realm.commitTransaction();
+
+        realm.beginTransaction();
+        for (ConferenceDay conferenceDay : conferenceDays) {
+            final RealmScheduleDayItemFilter item = realm
+                    .createObject(RealmScheduleDayItemFilter.class);
+            item.setActive(true);
+            item.setDayMs(conferenceDay.getDayMs());
+            item.setLabel(conferenceDay.getName());
+        }
+        realm.commitTransaction();
+
+        realm.close();
+    }
+
+    public void createTrackFiltersDefinition(List<RealmTrack> tracks) {
+        final Realm realm = realmProvider.getRealm();
+        realm.beginTransaction();
+        realm.allObjects(RealmScheduleTrackItemFilter.class).clear();
+        realm.commitTransaction();
+
+        realm.beginTransaction();
+        for (RealmTrack track : tracks) {
+            final RealmScheduleTrackItemFilter item = realm
+                    .createObject(RealmScheduleTrackItemFilter.class);
+            item.setActive(true);
+            item.setLabel(track.getTitle());
+            item.setTrackName(track.getId());
+        }
+        realm.commitTransaction();
+
+        realm.close();
+    }
+
+    public List<RealmScheduleTrackItemFilter> getTrackFilters() {
+        final Realm realm = realmProvider.getRealm();
+        final List<RealmScheduleTrackItemFilter> result
+                = realm.allObjects(RealmScheduleTrackItemFilter.class);
+        realm.close();
+        return result;
+    }
+
+    public List<RealmScheduleDayItemFilter> getDayFilters() {
+        final Realm realm = realmProvider.getRealm();
+        final List<RealmScheduleDayItemFilter> result
+                = realm.allObjects(RealmScheduleDayItemFilter.class);
+        realm.close();
+        return result;
+    }
+
+    public void updateFilter(RealmScheduleDayItemFilter itemFilter, boolean isActive) {
+        final Realm realm = realmProvider.getRealm();
+        realm.beginTransaction();
+        itemFilter.setActive(isActive);
+        realm.copyToRealmOrUpdate(itemFilter);
+        realm.commitTransaction();
+        realm.close();
+    }
+
+    public void updateFilter(RealmScheduleTrackItemFilter itemFilter, boolean isActive) {
+        final Realm realm = realmProvider.getRealm();
```

Comment:
jakby `RealmScheduleTrackItemFilter` i `RealmScheduleDayItemFilter` mialy wspolnego przodka powyzej `RealmItem` to moznaby to polaczyc, ale to opcjonalnie